deploy/chart/devfile-registry/templates/ingress.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "devfileregistry.fullname" . }}
+  labels:
+    app: {{ template "devfileregistry.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  annotations:
+    kubernetes.io/ingress.class: {{ .Values.global.ingress.class }}
+spec:
+  rules:
+  - host: {{ template "devfileregistry.hostname" . -}} . {{- .Values.global.ingress.domain }}
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: devfile-registry
+          servicePort: 8080
```

Comment:
Other than the index server service, do we want to expose `oci-metrics` and `index-metrics` services via the ingress as well?

Reply:
No, those should not be exposed publicly. Those are only exposed over the service to allow AlertManager and Prometheus on the cluster to scrape those endpoints.